client/sites/components/sites-dashboard.tsx

```diff
@@ -225,25 +231,6 @@ const SitesDashboard = ( {
 		if ( existingFields !== fieldsForBreakpoint ) {
 			setDataViewsState( ( prevState ) => ( { ...prevState, fields } ) );
 		}
-
-		const siteNameColumnWidth = getSiteNameColWidth( isDesktop, isWide );
-
-		if (
-			dataViewsState.type === 'table' &&
-			dataViewsState.layout?.styles?.site?.width !== siteNameColumnWidth
-		) {
-			setDataViewsState( {
-				...dataViewsState,
-				layout: {
-					styles: {
-						...dataViewsState.layout?.styles,
-						site: {
-							width: siteNameColumnWidth,
-						},
-					},
-				},
-			} );
-		}
```

Comment:
Doesn't seem to be doing much and the layout change was dropping combined fields on window resize. That can be fixed by adding the combined fields values here too but since the style change doesn't seem to be doing much I'm opting for removal.